CMakeLists.txt

```diff
@@ -61,9 +61,9 @@ if(BUILD_ANDROID)
 
     option(STRIP_ANDROID_LIBRARY "Strip the resulting android library" OFF)
 
-    # Set default API level to 21 if not configured explicitly
+    # Set default API level to 26 if not configured explicitly
     if(NOT ANDROID_PLATFORM)
-        set(ANDROID_PLATFORM "android-21")
+        set(ANDROID_PLATFORM "android-26")
```

Comment:
I'm not sure what this is intended to do since it only changes the default platform if none is specified, but this does not stop anyone from continuing to explicitly specify android-21. This doesn't change RenderDoc's minspec, all this does is means that CI and anyone building by default will no longer target the minspec correctly and it may hide problems.

If you want to build targeting a higher version you can, you can specify the `ANDROID_PLATFORM` parameter to cmake.

Reply:
Do we have place to specify android minspec for that? But anyway, I think I don't need it if I rewrite the code with your suggestions...

Reply:
Currently the min spec is Android 6 and only mentioned I think on the [documentation page](https://renderdoc.org/docs/how/how_android_capture.html) since the minspec is checked at runtime and shown when trying to connect to the device.

I don't know why you were changing this default so I can't answer if it's still needed if you're rewriting the code.